plutus-core/budgeting-bench/Bench.hs

```diff
@@ -13,18 +14,23 @@ import           Hedgehog
 import           Hedgehog.Internal.Gen
 import           Hedgehog.Internal.Tree
 import           Hedgehog.Range
-import           Language.PlutusCore
-import           Language.PlutusCore.Evaluation.Machine.Cek
+import           Language.PlutusCore                                        as PLC
 import           Language.PlutusCore.Evaluation.Machine.ExBudgetingDefaults
 import           Language.PlutusCore.Evaluation.Machine.ExMemory
+import           Language.PlutusCore.Generators.Interesting
 import           Language.PlutusCore.MkPlc
+import           Language.UntypedPlutusCore                                 as UT
+import           Language.UntypedPlutusCore.Evaluation.Machine.Cek
 import           System.Directory
 
 import           Criterion.Main
-import qualified Criterion.Types                                            as C
```

Comment:
There's a build failure saying 
```

budgeting-bench/Bench.hs:131:38: error:
    Not in scope: ‘C.csvFile’
    No module named ‘C’ is imported.
```
Is that because of this?